### PR TITLE
Move the TiSpark doc to the doc repository

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@andelf PTAL.

File: tispark/tispark-user-guide.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+For independent deployment of TiKV and the TiDB Connector for Spark, it is recommended to refer to the following recommendations
+ 
++ Hardware configuration
+ - For general purposes, please refer to the TiDB and TiKV hardware configuration [recommendations](https://github.com/pingcap/docs/blob/master/op-guide/recommendation.md#deployment-recommendations).
+ - If the usage is more focused on the analysis scenarios, you can increase the memory of the TiKV nodes to at least 64G. If using  Hard Disk Drive (HDD), it is recommended to use at least 8 disks.
+
++ TiKV parameters (default)
+
+```
+[Server]
```
Comment (andelf) [inline tispark/tispark-user-guide.md]: config file is wrong. should be lower-case here

**Comment by andelf** [comment]
Seem that whole content is copied from a rich word processing software like Word.

Extra spaces added in code blocks.

**Comment by queenypingcap** [comment]
@andelf PTAL.

File: tispark/tispark-user-guide.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+Running TiDB Connector for Spark on an existing Spark cluster does not require a reboot of the cluster. You can use Spark's `--jars` parameter to introduce the TiDB Connector for Spark as a dependency:
+
+```sh
+spark-shell --jars $PATH/tispark-0.1.0.jar
+```
+
+If you want to deploy TiDB Connector for Spark as a default component, simply place the TiDB Connector for Spark jar package into the jars path for each node of the Spark cluster and restart the Spark cluster:
+
+```sh
+$ {SPARK_INSTALL_PATH}/jars
```
Comment (andelf) [inline tispark/tispark-user-guide.md]: ${SPARK_INSTALL_PATH}/jars

File: tispark/tispark-user-guide.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
++---
+ +title: TiDB Connector for Spark User Guide
+ +category: user guide
++---
```
Comment (andelf) [inline tispark/tispark-user-guide.md]: plus sign here & above. should be removed

**Comment by queenypingcap** [comment]
@andelf PTAL again :)

**Pull request merged**